tests/integration/suite/test_rbac.py

```diff
@@ -761,6 +761,72 @@ def try_create_ns_secret():
     )
 
 
+def test_only_admin_can_crud_global_roles(admin_mc, user_mc, remove_resource):
```

Comment:
These two tests feel like they below under tests/integration/suite/test_global_roles.py.

Reply:
Realizing we have similar things in here already, it looks like it's becoming the `common` or `utils` of tests